app/src/store/playground/playgroundStore.tsx

```diff
@@ -251,6 +251,8 @@ export const createPlaygroundStore = (
     },
     setTemplateLanguage: (templateLanguage: TemplateLanguage) => {
       set({ templateLanguage });
+      // Re-compute variables when the template language changes
+      get().calculateVariables();
```

Comment:
I was literally thinking about this in bed last night that I forgot to put this here 😂